src/Ocelot/Request/Mapper/StreamHttpContent.cs

```diff
@@ -8,25 +8,24 @@ public class StreamHttpContent : HttpContent
     private const int DefaultBufferSize = 65536;
     public const long UnknownLength = -1;
     private readonly HttpContext _context;
+    private readonly long _contentLength;
```

Comment:
Ok, it's fine like this, but I would prefer using Headers.ContentLength. This value is set when calling AddContentHeaders class.

Reply:
And...